### ISSUE TITLE
Using other places lists with Google Takeout

### ISSUE THREAD

**Issue by Christine8691** [cga-wm/advgis-echo#29]
When using Google Takeout, are the starred places the only locations that can be saved to the JSON file to be brought into ArcGIS Pro or is it possible to create a new list (called whatever you named it) and use those locations?  If I wanted to create point layers in ArcGIS Pro of places on Google, if only starred places are able to be used, do I have to delete those I don't want included in the JSON file or delete them after the file is created?  For some applications, I would not want all of the starred locations in my download, but I also would not want to have delete them.   
![image](https://user-images.githubusercontent.com/63355555/140110333-9d0eb190-63d8-4e05-bb6b-1775709038ca.png)

**Comment by dt-woods** [comment]
My recommendation is to save places to one of your other lists, then try to export Takeout and see if there's a folder for it in the zip file.

**Comment by shwhitewm** [comment]
I would agree with @dt-woods .  And to your question yesterday in class about downloading points that are not in the "searchable places."   You could use the My Maps functionality of Google Maps.   In MyMaps it allows you to create a custom map.   Then you would go in and download that data of that map.  The same theory of the starred places and the downloading via Takeout but with a different part of the Google Maps features.     (Also I may not be this quick to respond in the future but Tyler and I have been working through the transition pieces of GitHub today).
